### PR TITLE
Add new marker for notebook tests

### PR DESCRIPTION
**Pull request by gaugup**
As a part of this change, also separating unit tests and notebooks tests. The notebook tests are failing for some reason which is not easy to fix. So separating the unit tests and notebooks tests to make PRs can merge if unit tests pass.

### CODE REVIEW

**Review by amit-sharma — APPROVED** [review]
Looks good!

File: requirements-test.txt [hunk 1]
```diff
@@ -7,4 +7,4 @@ twine
 pytest-mock
 torch
 # Pin scikit-learn
-scikit-learn<1.1.2
+# scikit-learn<1.1.2
```
Comment (amit-sharma) [inline requirements-test.txt:10]: I assume this change is because sklearn is already in requirements.txt?

Comment (gaugup) [reply requirements-test.txt]: I was trying to test if pin was required. Looks like it is still.

Comment (gaugup) [reply requirements-test.txt]: Removed this change.

**Pull request merged**